pkg/build/daggerbuild/arguments/docker.go

```diff
@@ -19,12 +19,12 @@ var (
 	}
 	AlpineImageFlag = &cli.StringFlag{
 		Name:  "alpine-base",
-		Usage: "The alpine image to use as the base image when building the Alpine version of the Grafana docker image",
+		Usage: "The image or image alias specified in the Dockerfile to be used as the base image when building the Alpine version of the Grafana docker image.",
```

Comment:
should we change the defaults here to alpine-base and ubuntu-base so they use the version from the dockerfile instead of `:latest`?